src/aktualizr_info/main.cc

```diff
@@ -43,9 +43,14 @@ int main(int argc, char **argv) {
     std::cout << "Storage backend: " << ((storage->type() == StorageType::kFileSystem) ? "Filesystem" : "Sqlite")
               << std::endl;
 
-    Uptane::RawMetaPack pack;
-
-    bool has_metadata = storage->loadMetadata(&pack);
+    std::string director_root;
+    std::string director_targets;
+    std::string images_root;
+    std::string images_targets;
+    bool has_metadata = storage->loadRole(&director_root, Uptane::RepositoryType::Director, Uptane::Role::Root());
```

Comment:
aktualizr_info is used during debugging to get an overview of the state of the system. `has_meta` indicates that aktualizr has already downloaded some meta from the server, and this process normally doesn't fail partially. I.e. if you have the connection, no problems with TLS certificates and the server is up, you will have all of needed metadata, otherwise you will have none.
The same tool allows you to show individual pieces of metadata if you want more detail.